transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/persistence/EventStore.java

```diff
@@ -39,8 +39,7 @@
   void recordSuccess(Iterable<PersistedEvent> events);
 
   /** Returns the timestamp when the backend is allowed to be called next time or null. */
```

Comment:
Please update the javadoc to indicate that we use zero for absense of value, not null.